src/main/java/racingCar/domains/Racing.java

```diff
@@ -0,0 +1,40 @@
+package racingCar.domains;
+
+import java.util.List;
+
+public class Racing {
+    private PositiveNumber rounds;
+    private PositiveNumber playedRound;
+    private Cars cars;
```

Comment:
```suggestion
public class Racing {
    private RoundsNumber rounds;
    private PlayedRoundNumber playedRound;
    private Cars cars;
```

복붙을 해서 코드의 중복이 발생하는 것처럼 보이지만, 저는 요구사항의 도메인 개념이 다르다면,
전혀 다른 객체로 선언하는 것을 좋아하는 편입니다.
왜냐하면 지금은 같은 로직을 사용하는 것처럼 보이더라도,
용어(그에 따른 개념)이 다르다면 미래에 언제든지 요구사항이 달라질 수 있기 떄문입니다.

사실 PositiveNumber 로 충분한데 진솔님이 워낙 잘해주셔서 마이너한 피드백을 드려보았습니다.
사실 여기까지 반영하는 건 좀 변태같기는 하네요.